app/common/renderer/components/Inspector/Inspector.jsx

```diff
@@ -92,23 +90,16 @@ const Inspector = (props) => {
     (screenshot && !screenshotError) ||
     (mjpegScreenshotUrl && (!isSourceRefreshOn || !isAwaitingMjpegStream));
 
-  // Calculate the ratio for scaling items overlaid on the screenshot
-  // (highlighter rectangles/circles, gestures, etc.)
-  const updateScaleRatio = (imgWidth) => setScaleRatio(windowSize.width / imgWidth);
-
-  const updateScaleRatioDebounced = debounce(updateScaleRatio, 500);
-
-  const updateSourceTreeWidth = () => {
-    // the idea here is to keep track of the screenshot image width. if it has
-    // too much space to the right or bottom, adjust the max-width of the
-    // screenshot container so the source tree flex adjusts to always fill the
-    // remaining space. This keeps everything looking tight.
-    if (!screenAndSourceEl.current) {
+  const updateScreenshotScale = () => {
+    // If the screenshot has too much space to the right or bottom, adjust the max width
+    // of its container, so the source tree always fills the remaining space.
+    // This keeps everything looking tight.
+    if (!screenshotContainerEl.current) {
       return;
     }
 
-    const screenshotBox = screenAndSourceEl.current.querySelector('#screenshotContainer');
-    const img = screenAndSourceEl.current.querySelector('#screenshotContainer img#screenshot');
+    const screenshotBox = screenshotContainerEl.current;
```

Comment:
should we use screenshotBox everywhere instead of screenshotContainerEl.current ?